capitalize_decorator.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'base_decorator'
+require_relative 'person'
+
+class CapitalizeDecorator < BaseDecorator
+  def initialize(nameable)
+    super
+    @nameable = nameable
+  end
+
+  def correct_name
+    super.upcase
```

Comment:
- [ ] We should `capitalize` the name, not make it all upper case. Kindly fix this.

  ```
  super.capitalize
  ```